.github/workflows/auto-assign-to-project.yml

```
@@ -0,0 +1,26 @@
+name: Auto Assign to Project(s)
+
+on:
+  pull_request:
+    types: [opened, labeled]
+env:
+  MY_GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+jobs:
+  assign_one_project:
```

Could you replace underscores with hyphens? Only to maintain consistency between workflows...

```suggestion
  assign-one-project:
```

---

Sure!